microsoft-edge/webview2/get-started/winui.md

```diff
@@ -73,9 +73,9 @@ To create a WebView2 app, start by creating a basic desktop project, to create a
 
 1. In **New Universal Windows Platform Project**, select the following values:
```

Comment:
2nd "installed" is superfluous & should be deleted:
Install the [WebView2 Runtime](https://developer.microsoft.com/microsoft-edge/webview2) or any [Microsoft Edge preview channel](https://www.microsoftedgeinsider.com/download) (Beta, Dev, or Canary) **installed** on Windows 10 version 1803 (build 17134) or later.

If we delete that word in this PR, note that this section wasn't updated to match pattern like https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2?branch=pr-en-us-2025#step-1---download-a-preview-channel-of-microsoft-edge